cmd/operator/deploy/operator/05-deployment.yaml

```diff
@@ -18,21 +18,21 @@ metadata:
   namespace: gmp-system
   name: gmp-operator
   labels:
+    app: managed-prometheus-operator
     app.kubernetes.io/name: gmp-operator
     app.kubernetes.io/component: operator
     app.kubernetes.io/part-of: gmp
 spec:
   replicas: 1
   selector:
     matchLabels:
```

Comment:
Can you add it back here and rerun `make presubmit`? It should regenerate the latest `examples/operator.yaml` with your change here.

Reply:
done.